portfolio.js

```diff
@@ -23,3 +23,169 @@ menuitems.forEach((item) => {
     close();
   });
 });
+
+const p1 = {
+  hname: 'Tonic',
+  pname: [' ', 'Canopy', 'Full Stack Dev', '2015'],
+  img: "<img id='card-image' src='images/one.png' alt=''/>",
+  description: 'daily selection of privately personalized reads; no accounts or sign-ups required.'
+ + 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Lorem ipsum dolor sit amet consectetur adipisicing elit.'
+ + 'Odit exercitationem quidem eius illum',
+  tech: [' ', 'html', 'Ruby on rails', 'css', 'javaScript'],
+  link: '#',
+  psource: '#',
+};
+const p2 = {
+  hname: 'Multi-Post Stories',
+  pname: [' ', 'Canopy', 'Full Stack Dev', '2015'],
+  img: "<img id='card-image' src='images/two.png' alt=''/>",
+  description: 'daily selection of privately personalized reads; no accounts or sign-ups required.'
+ + 'Lorem ipsum dolor sit amet consectetur adipisicing elit.Lorem ipsum dolor sit amet consectetur adipisicing elit.'
+  + 'Odit exercitationem quidem eius illum',
+  tech: [' ', 'html', 'Ruby on rails', 'css', 'javaScript'],
+  link: '#',
+  psource: '#',
+};
```

Comment:
- [ ] I see that you have created the projects individually which is why you have repetitive functions handling them separately, but it will be better to make the projects an array of objects that will be handled with one function. You will just have to loop through the array to create each project and its modal.